docs/aqt/access.md

```diff
@@ -1,28 +1,43 @@
 # Access and Authentication
 
-AQT offers access to several quantum computing devices, so-called backends, ranging from real-hardware ion traps with various number of ions to quantum computing simulators including different noise models. To get an overview of available devices visit [www.aqt.eu](https://www.aqt.eu) and get direct access to the devices via the [AQT gateway portal](https://api-gateway.aqt.eu).
+AQT offers access to several quantum computing devices, called backends,
+ranging from real-hardware ion traps with various number of ions to
+quantum computing simulators including different noise models.
+To get an overview of available devices visit
+[www.aqt.eu](https://www.aqt.eu) and get direct access to the devices via the
```

Comment:
Add `{:.external}` to external links per https://www.tensorflow.org/community/contribute/docs_style

Below as well.

Reply:
Good catch.  I need to remember this!  Done.